windows/hcs/hcs.go

```diff
@@ -292,6 +292,17 @@ func (c *Container) Delete(ctx context.Context) {
 	}
 }
 
+func (c *Container) RemoveProcess(pid uint32) {
+	c.Lock()
+	defer c.Unlock()
+	for i := range c.processes {
+		if c.processes[i].pid == pid {
+			c.processes = append(c.processes[:i], c.processes[i+1:]...)
```

Comment:
This is already done by the go routine started when the process is created (https://github.com/containerd/containerd/blob/master/windows/hcs/hcs.go#L407)